docs/docs/developer/api.md

```diff
@@ -0,0 +1,13 @@
+# API
+
+<!-- dprint-ignore-start -->
+
+::: kpops.api
+
+<!-- dprint-ignore-end -->
+
+<!-- dprint-ignore-start -->
+
+::: kpops.pipeline.Pipeline
```

Comment:
I don't really the `Pipeline` class should be in the public API docs.

Reply:
Removed

Reply:
yes, `Pipeline` can be made public imo